game/views.py

```
@@ -17,15 +18,24 @@
 from django.contrib.auth.decorators import login_required
 
 from .engine import ChessGame
-from .models import GameResult
+from .models import GameResult, PrivateRoom
 from game.services import cleanup_stale_games
 
+def generate_room_code(length=8):
+    characters = string.ascii_uppercase + string.digits
+    
+    while True:
+        code = ''.join(random.choices(characters, k=length))
```

Ruff (0.15.13)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)